osquery/logger/CMakeLists.txt

```diff
@@ -18,6 +18,17 @@ else()
   ADD_OSQUERY_LINK_ADDITIONAL("aws-cpp-sdk-core")
 endif()
 
+if(NOT WINDOWS)
+  ADD_OSQUERY_LINK_ADDITIONAL("z")
+  ADD_OSQUERY_LINK_ADDITIONAL("pthread")
+
+  if(APPLE)
+    ADD_OSQUERY_LINK_ADDITIONAL("${BUILD_DEPS}/lib/librdkafka.dylib")
```

Comment:
There is, but it wasn't able to successfully build against it.  I forgot what the exact issue was.  I'll check and update on this.